website/content/docs/secrets/pki/rotation-primitives.mdx

```diff
@@ -1,7 +1,8 @@
 ---
 layout: docs
-page_title: 'PKI - Secrets Engine: Rotation Primitives'
-description: The PKI secrets engine for Vault generates TLS certificates.
+page_title: 'PKI secrets engine - rotation primitives'
+description: >-
+   Understand the X.509 certificate primitive fields when you are using the PKI secrets engine.  
```

Comment:
I don't think "primitive fields" is correct. Primitives here refer to certificate types, so maybe something along the lines of:
>Understand the primitive certificate types for root and intermediate CA rotation.